config/services.yaml

```diff
@@ -11,6 +11,9 @@ services:
         resource: '../src/Surfnet/AzureMfa/Infrastructure/*'
         exclude: '../src/Surfnet/AzureMfa/Infrastructure/{DependencyInjection,Entity,Migrations,Tests,Kernel.php}'
 
+    Surfnet\AzureMfa\Application\:
```

Comment:
Is it possible to combine both entries (application and domain) ?

Reply:
Do you mean by defining `Surfnet\AzureMfa` instead of adding two entries for both Application and Domain? That could work, but I wonder if this would interfere with the infrastructure config.

Reply:
I was wondering that too and I wanted to know if you tried.

Reply:
I tried, and committed 136e39c just now